examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1705,10 +1706,12 @@ class GLTFMeshQuantizationExtension {
 // Specification: https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#appendix-c-spline-interpolation
 class GLTFCubicSplineInterpolant extends Interpolant {
 
-	constructor( parameterPositions, sampleValues, sampleSize, resultBuffer ) {
+	constructor( parameterPositions, sampleValues, sampleSize, resultBuffer, targetQuaternion ) {
```

Comment:
I think a better parameter name would be `normalize` -- or `normalizeOutput`.

Reply:
Yeah, thought about this, too. But having a parameter that has `quaternion` in its name sounds more descriptive to me. And thus easier to understand.

Reply:
It is a bit confusing IMO. It is not a  "target" quaternion. It is a boolean.

Furthermore, using "quaternion" is too restrictive. You can interpolate positions on a sphere, for example.

Using the boolean parameter `normalize` follows the pattern used elsewhere in the library.

Reply:
The idea that the instance of `GLTFCubicSplineInterpolant` targets a rotation property and thus a quaternion.

Besides, the parameter only affects quaternions and thus a generic name is misleading.

Reply:
Changed the name to `normalizeQuaternion`.

Reply:
> If the method is only used to interpolate quaternions,

No it isn't.

> If the method can be used to interpolate something other than a quaternion, then normalizeQuaternion makes no sense. A better name for the parameter would be normalize.

The normalization is only necessary for quaternions.

Reply:
I can imagine hypothetical situations where a user might want to interpolate normalized vectors, but glTF does not include any such cases. Since this is an internal API I don't think I have any preference on the naming, and implementing unused normalization of Vector2, Vector3, is certainly overkill. One other idea (again, previous solutions are also fine) would be to use a simple subclass instead of a constructor parameter:

```js
class GLTFCubicSplineQuaternionInterpolant extends GLTFCubicSplineInterpolant {
  /* overrides interpolate_ ... */
}
```

Reply:
>use a simple subclass instead of a constructor parameter

That sounds like the best idea to me.